src/rgw/rgw_file.h

```diff
@@ -740,7 +740,7 @@ namespace rgw {
   class RGWLibFS
   {
     CephContext* cct;
-    struct rgw_fs fs;
```

Comment:
> consistency: use ctor-initializer syntax where already used?

@mattbenjamin Could you please clarify the source of this text ? Is it a warning ?

Yes, we can use the ctor-initializer and fix it for now. But what if another new constructor in the future introduce the same issue ?

So I did this fix based on this: http://en.cppreference.com/w/cpp/language/value_initialization